client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -555,6 +556,33 @@ static Request rollover(RolloverRequest rolloverRequest) throws IOException {
         return new Request(HttpPost.METHOD_NAME, endpoint, params.getParams(), entity);
     }
 
+    static Request getAlias(GetAliasesRequest getAliasesRequest) {
+        Params params = Params.builder();
+        params.withIndicesOptions(getAliasesRequest.indicesOptions());
+        params.withLocal(getAliasesRequest.local());
+        if (false == CollectionUtils.isEmpty(getAliasesRequest.aliases())) {
+            params.withName(getAliasesRequest.aliases());
+        } else {
+            params.withName(getAliasesRequest.name());
+        }
+        String endpoint = endpoint(optional(getAliasesRequest.indices(), "_all"), "_alias");
```

Comment:
If there are no indices and no aliases specified ( `/_alias` ), then `GetAllAliases` action will triggered ( which returns a different response ). By using `_all` when there are no indices, we can use the `GetAliases` instead and get the same results ;)